Open Judge System/OJS.Services/Data/ParticipantScores/IParticipantScoresDataService.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace OJS.Services.Data.ParticipantScores
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using OJS.Data.Models;
+    using OJS.Services.Common;
+
+    public interface IParticipantScoresDataService : IService
+    {
+        ParticipantScore Get(int participantId, int problemId);
+
+        ParticipantScore Get(int participantId, int problemId, bool isOfficial);
+
+        IQueryable<ParticipantScore> GetAll();
```

Comment:
here you must have `Query` as suffix, because it is data service and you return query